com.richwhitehouse.BigPEmu.yaml

```diff
@@ -0,0 +1,41 @@
+id: com.richwhitehouse.BigPEmu
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: bigpemu
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.bigpemu_userdata
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
make up your mind. Does it support wayland or doesn't it not? There are options here exclusive to each other.

Reply:
I see now, it does support wayland, so removed `socket=x11` and keeping `socket=fallback-x11`